### ISSUE TITLE
Color contrast ratio of the keyboard focus present on the menu items present in "output" window is less than 3:1: A11y_VisualStudioCodeClient_Output_Non-textContrast

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#285525]
Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Feng.ms%2Fdocs%2Fcoreai%2Fdevdiv%2Fone-engineering-system-1es%2F1es-docs%2Fes-chat%2Findex&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221175207%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=ju8Raw6mIhT5G3gWbZSkS0CAN2zkv2ycA5ifSSqoxOw%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fchannel%2F19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2FCAI%2520Accessibility%2520Support%2520Channel%3FgroupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221198042%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=DCNVsZe8oDZWqm97AI0ee4OoWWecR6zw6qlhTuzP7MI%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Foutlook.office.com%2Fbook%2Fmicrosoft2779007%2540microsoft.onmicrosoft.com%2Fs%2FbLeGBltMW0qI0WRLsTqx2A2%3Fismsaljsauthenabled&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221214666%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=CnCCdDgEmll2oUQUdMjXusAgV8PneqZVYvzgxAfk7Ik%3D&reserved=0).

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier." 

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".

GitHub Tags: 
#A11yTCS;#Visual Studio Code Client;#E2E_VisualStudioCodeClient_Win32_Dec2025;#DesktopApp;#FTP;#Win32;#A11ySev2;#A11yMAS;#WCAG1.4.11; #Non-textContrast;

Environmental Details: 
App Name: Visual Studio Code Client 
#URL: Visual Studio Code Insider
Microsoft Windows 11 Enterprise 
OS Build:  26200.7392

Repro Steps: 
1. Open Visual Studio Code Insiders editor
2. TAB till combo box present in "Output" window and press ENTER.
3. Open accessibility insights for windows and check color contrast ratio of the control.
4. Observe the issue Color contrast ratio of the keyboard focus present on the menu items present in "output" window is less than 3:1.

Actual Results: 
Color contrast ratio of the keyboard focus present on the menu items present in "output" window is less than 3:1.

Expected Results: 
The color contrast ratio of the text with the background should be greater than or equal to 3:1.

User Impact: 
Users with visual impairments or low vision may find it difficult to read and distinguish text due to insufficient contrast. This can hinder their ability to access content effectively, creating usability challenges and reducing accessibility.

Attachments:

<img width="1289" height="626" alt="Image" src="https://github.com/user-attachments/assets/ce3c445f-d958-49a4-9026-5b952b04f50e" />

**Comment by kupatkar99** [comment]
When focus is landing on the list items users are able to differentiate the selected and non selected list items and also color contrast ratio with the selected background is greater than 4.5:1. Hence, closing the bug. 
#Closed; 

<img width="1353" height="685" alt="Image" src="https://github.com/user-attachments/assets/1c3a2552-f3ba-441b-b875-dd44002826fd" />
